### PR TITLE
added FTL version to log output

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

**Comment by nkubala** [comment]
can you add a git tag at this commit so we don't forget? if we don't set through a Makefile or something we should have some way to mark where we bumped the version each time.

**Comment by aaron-prindle** [comment]
there are git tags for the versions, they are released by a git tag cloudbuild trigger

**Comment by aaron-prindle** [comment]
is that what you are referring to? it is confusing but our build system is git tag based so manually syncing constants.py -- git tags is the solution for now

**Comment by nkubala** [comment]
ah yeah ok that's more or less what I was talking about. just want something embedded in git so we don't have to keep track of it ourselves

File: ftl/common/constants.py [hunk 1]
```diff
@@ -20,6 +20,9 @@
 # docker transport thread config
 THREADS = 32
 
+# ftl version
+FTL_VERSION = "v0.2.0"
```
Comment (squee1945) [inline ftl/common/constants.py]: Is this part of the cache key?

**Pull request merged**